web_pwa_cache/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,36 @@
+Odoo Client Proxy
+
+This module is used to add support to cache POST/GET requests. Can handle Odoo requests and responses to give an offline experience.
+
+Cache Types
+~~~~~~~~~~~
+
+- Model -> Gets records, related actions, views, default values and filters
+- Client QWeb -> Gets indepent client views (used by widgets)
+- function -> Cache function calls
+- Onchange -> Cache onchange calls
+- Onchange with formula -> Cache onchange calls using formulas
+- Post -> Cache indepent POST requests (used by widgets)
+- Get -> Cache indepent GET requests
+
+How Works
+~~~~~~~~~
+
+The module handle two modes:
+
+- Normal: Runs Odoo as normal
+- Standalone: When the app starts you will request to set the mode (offline or online).
+    - Online: All requests except Create, Update and Delete operations goes from cache first, if not found, tries from network.
+    - Offline: All requests goes from/to cache
```

Comment:
This 'Online mode' still breaks the onchanges and other small things; would it make sense to separate the modes into 'Online Cached' and 'Fully Online'? Then the user can choose if he wants to live with the glitches and save bandwidth, or go for full functionality like in a browser.

Reply:
Online mode should not break anything... when you call to an onchange/resource that is not cached the request goes to Odoo server.
I will add it to the roadmap, but in principle it is not a functionality we are looking for on our side.

Reply:
I think what broke was the request that used the `?=` operator - will watch for any other things breaking on Online mode.